### ISSUE TITLE
Add support for arabic/urdu fonts

### ISSUE THREAD

**Issue by recluzegeek** [milgra/vmp#2]
### Description
- [Paste Bin Logs](https://pastebin.com/FGCdhSsa)
> Appreciate your efforts in making this player. 

- I've old tracks that have metadata in arabic/urdu and they're in large number(so cannot change them manually). vmp doesn't seems to render their metadata correctly. Refer below

![issue](https://github.com/milgra/vmp/assets/72850566/ab58eae0-6852-4582-ad63-9a5a6d82ca59)

- Used this command:
`vmp -l /mnt/win/Music --font=/usr/share/fonts/noto/NotoNaskhArabicUI-Regular.ttf`

and tested with these fonts
- /usr/share/fonts/noto/NotoSansArabic-Light.ttf: Noto Sans Arabic,Noto Sans Arabic Light:style=Light,Regular
- /usr/share/fonts/noto/NotoNaskhArabicUI-Bold.ttf: Noto Naskh Arabic UI:style=Bold
- /usr/share/fonts/noto/NotoSansArabic-Black.ttf: Noto Sans Arabic,Noto Sans Arabic Black:style=Black,Regular
- /usr/share/fonts/noto/NotoNaskhArabic-Medium.ttf: Noto Naskh Arabic,Noto Naskh Arabic Medium:style=Medium,Regular
- /usr/share/fonts/noto/NotoSansArabic-Bold.ttf: Noto Sans Arabic:style=Bold
- /usr/share/fonts/noto/NotoNaskhArabicUI-Medium.ttf: Noto Naskh Arabic UI,Noto Naskh Arabic UI Medium:style=Medium,Regular
- /usr/share/fonts/noto/NotoSansArabic-Regular.ttf: Noto Sans Arabic:style=Regular
- /usr/share/fonts/noto/NotoNastaliqUrdu-Bold.ttf: Noto Nastaliq Urdu:style=Bold
- /usr/share/fonts/noto/NotoNastaliqUrdu-Regular.ttf: Noto Nastaliq Urdu:style=Regular
- /usr/share/fonts/noto/NotoSansArabic-Thin.ttf: Noto Sans Arabic,Noto Sans Arabic Thin:style=Thin,Regular
- /usr/share/fonts/noto/NotoNaskhArabic-Bold.ttf: Noto Naskh Arabic:style=Bold
- /usr/share/fonts/noto/NotoNaskhArabic-Regular.ttf: Noto Naskh Arabic:style=Regular
- /usr/share/fonts/noto/NotoNaskhArabicUI-Regular.ttf: Noto Naskh Arabic UI:style=Regular
- /usr/share/fonts/noto/NotoSansArabic-Medium.ttf: Noto Sans Arabic,Noto Sans Arabic Medium:style=Medium,Regular

**Comment by milgra** [comment]
Hi! Thank you for using vmp! This issue is very strange, unicode fonts should work out of the box, maybe reading out the id3 tags has some problems. Does ffmpeg -i xy.mp3 or ffmpeg -i xy.mp3 -f ffmetadata in.txt work on these files and do they contain correct fonts?

**Comment by recluzegeek** [comment]
Yeah ffmpeg is working on the files. Here's the content of in.txt.
 
```
└──| cat in.txt 
;FFMETADATA1
album=مصحف الشيخ عبدالرحمن السديس
artist=عبدالرحمن بن عبدالعزيز السديس
comment=jam3-fawaed.blogspot.com
genre=قرآن كريم
title=مصحف الحرم المكي الشريف 
date=1436
encoder=Lavf60.3.100
```
![ffmpeg](https://github.com/milgra/vmp/assets/72850566/cebcaffe-1808-43d2-b499-4c35a9b6740b)

**Comment by milgra** [comment]
Cool, thank you, I will look into it.
